deleting gnu from "tech I like"

stefanasandei opened the pull request:
gnu is bloat

stefanasandei committed:
deleting gnu from "tech I like"

gnu is bloat

Dimitri1998 commented:
I did a quick code review, should not break anything in production and it looks fine.
So yeah, @michaelradu  you can approve it.

stefanasandei committed:
Removing java

I want to guide you down the right path, ur not a masochist. You don't like java. Together we can help you.
